integrations/keras/prune_resnet20.py

```diff
@@ -74,16 +76,33 @@ def download_model_and_recipe(root_dir: str):
         repo="sparseml",
         dataset="cifar_10",
         training_scheme=None,
-        optim_name="pruned",
-        optim_category="conservative",
+        optim_name="base",
+        optim_category=None,
         optim_target=None,
         override_parent_path=model_dir,
     )
-    zoo_model.download()
-    model_file_path = zoo_model.framework_files[0].downloaded_path()
+    base_zoo_model.download()
+    model_file_path = base_zoo_model.framework_files[0].downloaded_path()
     if not os.path.exists(model_file_path) or not model_file_path.endswith(".h5"):
         raise RuntimeError("Model file not found: {}".format(model_file_path))
-    recipe_file_path = zoo_model.recipes[0].downloaded_path()
+
+    # Load recipe to use
+    pruned_zoo_model = Zoo.load_model(
+        domain="cv",
+        sub_domain="classification",
+        architecture="resnet_v1",
+        sub_architecture=20,
+        framework="keras",
+        repo="sparseml",
+        dataset="cifar_10",
+        training_scheme=None,
+        optim_name="pruned",
+        optim_category="conservative",
+        optim_target=None,
+        override_parent_path=model_dir,
+    )
+    pruned_zoo_model.download()
```

Comment:
Why are we downloading the whole model if we're only using the recipe? Also, @kevinaer / @bfineran we added in a way to get the base model from a pruned model, right?